client/src/components/admin/ResultAttemptsForm/ResultAttemptsForm.jsx

```diff
@@ -91,15 +92,23 @@ function ResultAttemptsForm({
       const attempts = trimTrailingSkipped(attemptResults).map((result) => ({
         result,
       }));
-      onSubmit(attempts);
+      onSubmit(attempts, result.person);
     });
   }
 
   function confirmSubmission() {
+    // We don't want to show a duplicate warning if the user is submitting
+    // already-entered results for the correct competitor.  This is often used
+    // as a quick way to refresh data without refreshing the whole page.
+    const resultsToCheck = combinedResults.filter(
+      (res) => res.id !== result.id,
+    );
```

Comment:
I would compute `combinedResults` together with the filter here.

Computing them upfront, as in the current version, doesn't have any benefit, because it would be recomputed after every submission anyway.